module/zfs/zcp.c

```diff
@@ -791,19 +776,32 @@ zcp_lua_alloc(void *ud, void *ptr, size_t osize, size_t nsize)
 static void
 zcp_lua_counthook(lua_State *state, lua_Debug *ar)
```

Comment:
Maybe this function name could be updated to reflect the fact that it's no longer just counting instructions, but also checking for cancellation?